TenzoMath/Source/StrainGauge.h

```diff
@@ -0,0 +1,85 @@
+﻿#ifndef STRAIN_GAUGE_H
+#define STRAIN_GAUGE_H
+
+#include <array>
+
+#include <Windows.h>
+
+
+/**
+* \brief Main class of communication between user and strain gauge.
+*/
+class StrainGauge
+{
+    /**
+    * \brief Declaration of serial port.
+    */
+    HANDLE _hSerial{};
+
+    /**
+    * \brief Name of the serial port in the form: L"COM5".
+    */
+    LPCWSTR _sPortName;
+
+    /**
+    * \brief Size of buffer for sending data.
+    */
+    DWORD _iSize{};
+
+    /**
+    * \brief Recieved code from the strain gauge.
+    */
+    int16_t _sReceivedChar{};
+
+    /**
+    * \brief Received code with information about the data packet.
+    */
+    char _skip{};
+
+    /**
+    * \brief Array of chars for sensor with info to get data packet.
+    */
+    char _sSendChar[5];
+
+    /**
+    * \brief Function of initializing of COM port.
+    * \return True if successful.
+    */
+    bool initializeCom();
+
+public:
+    /**
+    * \brief Constructor of Strain gauge with port of connection.
+    * \param[in] port Name of the port with sensor in the form: L"COM5".
+    */
+    explicit StrainGauge(const LPCTSTR port);
+
+    /**
+    * \brief Function of the connect to the strain gauge.
+    * \return True if successful.
+    */
+    bool connect();
```

Comment:
**(note)**: A constructor should create a fully initialized object.
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rc-complete